client/src/components/form/Dropdown/Dropdown.scss

```diff
@@ -56,7 +56,7 @@
   border-radius: 6px;
   margin-top: 0.2em;
   position: absolute;
-  z-index: 100;
+  z-index: 9;
```

Comment:
adjusted the z-index so that it wouldn't go over the navbar -- will adjust z-index of navbar and footer tonight and make it greater!
![image](https://user-images.githubusercontent.com/86009011/175395629-5c1d4d70-3dcc-4595-a414-c600a81a9515.png)